tools/cloudflow-operator/src/main/scala/cloudflow/operator/action/CloudflowStatus.scala

```diff
@@ -170,11 +175,25 @@ object CloudflowStatus {
 
   private def createStreamletStatuses(spec: App.Spec, runners: Map[String, Runner[_]]) =
     spec.deployments.map { deployment =>
-      val expectedPodCount = runners.get(deployment.runtime).map(_.expectedPodCount(deployment)).getOrElse(1)
-      App.StreamletStatus(
-        streamletName = deployment.streamletName,
-        expectedPodCount = Some(expectedPodCount),
-        podStatuses = Nil)
+      if (deployment.runtime == FlinkRunner.Runtime && !runners.contains(FlinkRunner.Runtime)) {
+        App.StreamletStatus(
+          streamletName = deployment.streamletName,
+          expectedPodCount = Some(0),
+          podStatuses = Seq(
+            App.PodStatus(
+              name = "<external>",
```

Comment:
why not depend on this `"<external>"` string to identify the podstatus that should be treated differently?